blue-green-deploy/README.md

```diff
@@ -0,0 +1,87 @@
+# Blue Green Deploy on Kubernetes
+
+This task gives you a flexibility to deploy the newer version of the application in the green zone without disturbing the previous version by making the service point to the newer version of the application deployed. It gives a rapid way to rollback - if anything goes wrong you switch the router back to your blue environment.
+
+  - If you are deploying version `v1` of the application then you need to provide deployment and service manifests via the `ConfigMap` and in the `params` the service name and version. Sample Kubernetes  manifests for version v1 can be found [here](./v1-deploy).
+  - If you are deploying version `v2` of the application the you need to provide only the deployment manifest and in the `params` the new deployment service name and version. Sample Kubernetes  manifests for version v2 can be found [here](./v2-deploy).
+
+![](./blue_green_deployments.png)
```

Comment:
What is this diagram refer too ? I see some green and blue blobs and arrow between the green blobs but i am not sure what is trying to explains!